### ISSUE TITLE
Zero loss in case lora qwen3-4b-reranker tuning

### ISSUE THREAD

**Issue by thekirillisupov** [modelscope/ms-swift#4820]
```
nproc_per_node=4
# 4*47G
# losses: plugin/loss.py
NPROC_PER_NODE=$nproc_per_node \
swift sft \
    --model Qwen/Qwen3-Reranker-4B \
    --task_type generative_reranker \
    --loss_type listwise_generative_reranker \
    --train_type lora \
    --lora_rank 8 --lora_alpha 32 \
    --lora_dropout 0.1 \
    --target_modules q_proj v_proj \
    --dataset  MTEB/scidocs-reranking \
    --split_dataset_ratio 0.05 \
    --eval_strategy steps \
    --output_dir output \
    --eval_steps 10 \
    --num_train_epochs 3 \
    --save_strategy "epoch" \
    --per_device_train_batch_size 2 \
    --per_device_eval_batch_size 2 \
    --gradient_accumulation_steps 8 \
    --learning_rate 2e-4 \
    --label_names labels \
    --warmup_ratio 0.01 \
    --max_grad_norm 1.0 \
    --dataloader_drop_last true
```

<img width="858" alt="Image" src="https://github.com/user-attachments/assets/a0c45a87-8d94-44b4-9405-33d5c20e13d0" />

**Comment by 0russwest0** [comment]
I tested with the provided script, and the loss is normal. Your issue might be caused by https://github.com/modelscope/ms-swift/pull/4795. Try pulling the latest version to resolve the problem.

![Image](https://github.com/user-attachments/assets/27dfb589-0097-4460-afb8-1e0de6d55c07)

**Comment by thekirillisupov** [comment]
Thx! works after clear caching
